packages/core/test/init.test.ts

```diff
@@ -186,6 +186,23 @@ describe('init:', () => {
 			},
 		})
 	})
+
+	test('addModel works when it was used alone', () => {
```

Comment:
move this test better to `store.test.ts` please, we already have there some tests related to storeModel =)